app/helpers/home_helper.rb

```diff
@@ -1,2 +1,13 @@
 module HomeHelper
+  def get_team_percentage
+    total_projects = Project.all.count
```

Comment:
Project.all is happening twice in the same view - in this function and the next one. Refactor the code so that we load projects just once from database and apply queries and filtering there. We can still have a helper method to do calculations but queries can be moved to the controller.